app/components/profile/EventCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React = require('react');
+import { Card } from 'react-native-paper'
+import { StyleSheet, Text, View } from 'react-native';
+import { EventData } from "../../types/EventData";
+
+const EventCard = (props: { eventData: EventData }) => {
+    const styles = StyleSheet.create({
+        eventCard: {
+            width: "100%",
+            flexWrap: 'wrap',
+            flexDirection: 'row',
+            padding: "4%",
+        },
+
+        eventTitle: {
+            width: "75%",
+        },
+
+        eventDate: {
+            width: "25%"
+        },
+
+        eventLocation: {
+            width: "58%"
+        },
+
+        eventTime: {
+            width: "42%"
+        }
+    });
+
+
+    return (
+        <Card style={{ margin: 15 }}>
+            <View style={styles.eventCard}>
+                <View style={styles.eventTitle}>
+                    <Text
+                        numberOfLines={1}
+                        style={{ fontSize: 25, textAlign: 'left' }}
+                    >
+                        {props.eventData.title}
+                    </Text>
+                </View>
+                <View style={styles.eventDate}>
+                    <Text
+                        numberOfLines={1}
+                        style={{ fontSize: 20, textAlign: 'right' }}
+                    >
+                        {props.eventData.date.getMonth()+1}/{props.eventData.date.getDate()}
```

Comment:
Can you replace this with `{props.eventData.date.toLocaleDateString(undefined, {month: "numeric", day: "numeric"})}`? This should format the date string correctly for different locales.

Reply:
ok!